### PR TITLE
CE-224: minor lintage

### PR DESCRIPTION
**Pull request by ialarmedalien**
Removing an unused file and fixing a few very minor linting issues.

### CODE REVIEW

File: lib/StaticNarrative/narrative/narrative_util.py [hunk 1]
```diff
@@ -125,14 +124,14 @@ def get_static_info(ws_url: str, token: str, ws_id: int) -> Dict:
         raise WorkspaceError(err, ws_id)
 
     info = {}
-    meta = ws_info[8]
-    if "static_narrative_ver" in meta:
+    ws_meta = ws_info[8]
```
Comment (ialarmedalien) [inline lib/StaticNarrative/narrative/narrative_util.py:127]: edited these to be in sync with `fetch_test_narrative.py` below

File: lib/StaticNarrative/narrative/updater.py [hunk 2]
```diff
@@ -1,499 +0,0 @@
-"""
```
Comment (ialarmedalien) [inline lib/StaticNarrative/narrative/updater.py:1]: nothing from this file gets used ==> bye bye!

Comment (briehl) [reply lib/StaticNarrative/narrative/updater.py:1]: Yeah, I remember putting that in there to try to auto-update from a possibly older Narrative object version that a user might try to make a Static Narrative from. Then I guess I just figured on 1. avoiding code creep from copy-pasta, and 2. letting the Static process just crash and tell the user they're using a very old Narrative and they should make a new save first. And never got around to deleting this.

**Review by briehl — APPROVED** [review]
👍 🎉 🚀 😹 and all other nonspecifically positive emojis.

**Pull request merged**